test/built-ins/Proxy/getOwnPropertyDescriptor/result-type-is-not-object-nor-undefined.js

```diff
@@ -42,7 +41,3 @@ assert.throws(TypeError, function() {
 assert.throws(TypeError, function() {
   Object.getOwnPropertyDescriptor(p, "boolean");
 });
-
-assert.throws(TypeError, function() {
-  Object.getOwnPropertyDescriptor(p, "fn");
-});
```

Comment:
good catch! although, this will throw a TypeError later anyway while comparing the descriptors as the proxy trap can't inform a non configurable descriptor if the target property is configurable. In other words, this assertion is still correct but in another file.

I'll check if we have a test for that.

```
17. If resultDesc.[[Configurable]] is false, then
  a. If targetDesc is undefined or targetDesc.[[Configurable]] is true, then
    i. Throw a TypeError exception.
```